### PR TITLE
[WIP] Adding Jira wrapper for test_vm_create

### PR DESCRIPTION
**Pull request by jan-zmeskal**
{{pytest: cfme/tests/cloud_infra_common/test_events.py::test_vm_create --use-provider rhv41 -v}}

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_events.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def vm_crud(provider, setup_provider_modscope, small_template_modscope):
 
 
 @pytest.mark.uncollectif(BZ(1491576, forced_streams=['5.7']).blocks, 'BZ 1491576')
-@pytest.mark.meta(blockers=[1238371], automates=[1238371])
+@pytest.mark.meta(blockers=[JIRA('RHCFQE-4749')])
```
Comment (rlbabyuk) [inline cfme/tests/cloud_infra_common/test_events.py]: **Required/Question**: Does this issue touch only rhv41 provider?

Comment (jan-zmeskal) [reply cfme/tests/cloud_infra_common/test_events.py]: It's really hard for me to say since I do not work with any other provider.

Comment (izapolsk) [reply cfme/tests/cloud_infra_common/test_events.py]: yes, it is relevant only for rhv41. Please don't put Jira. I'm raising BZ and will add it to test soon

Comment (jan-zmeskal) [reply cfme/tests/cloud_infra_common/test_events.py]: @izapolsk alrgiht, this PR is not really required, so closing. Thank you for looking into it so quickly

**Pull request closed without merging**